pkg/clients/compute/aks.go

```diff
@@ -119,40 +78,26 @@ func (c AggregateClient) GetManagedCluster(ctx context.Context, ac *v1alpha3.AKS
 
 // EnsureManagedCluster ensures the supplied AKS cluster exists, including
 // ensuring any required service principals and role assignments exist.
```

Comment:
It would be good to update this comment since we're no longer ensuring SP creation and its role assignments.

Reply:
Thank you @ezgidemirel. Removed mentions of the service principals and role assignments as we are no longer dealing with them.